iset.mm

```diff
@@ -133932,11 +133932,17 @@ C_ if ( A. k e. suc J ( Q ` ( i e. _om
   ${
     $d x y $.
     $( The Schroeder-Bernstein Theorem is equivalent to excluded middle.  This
-       is Metamath 100 proof #25.  The Metamath Proof Explorer (MPE) database
-       uses classical logic, which accepts excluded middle as an axiom.  Thus
-       the Schroeder-Bernstein Theorem is true in MPE; see
-       ~ https://us.metamath.org/mpeuni/sbth.html .  (Contributed by Jim
-       Kingdon, 13-Aug-2022.) $)
+       is Metamath 100 proof #25.  The forward direction ( ~ isbth ) is the
+       proof of the Schroeder-Bernstein Theorem from the Metamath Proof
+       Explorer database (in which excluded middle holds), but adapted to use
+       ` EXMID ` as an antecedent rather than being unconditionally true.  The
+       Metamath Proof Explorer proof can be found at
+       ~ https://us.metamath.org/mpeuni/sbth.html .
```

Comment:
```suggestion
       Explorer database (in which excluded middle holds), but adapted to use
       ` EXMID ` as an antecedent rather than being unconditionally true, as in
       the non-intuitionist proof
       ~ https://us.metamath.org/mpeuni/sbth.html .
```

Reply:
Suggestion accepted.

Reply:
I would have written "classical proof", but this is fine by me.